<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="initial_x" default="0" />
    <arg name="initial_y" default="7" />
    <arg name="des_x" default="0" />
    <arg name="des_y" default="-3" />
    <param name="initial_x" value="$(arg initial_x)" />
    <param name="initial_y" value="$(arg initial_y)" />
    <param name="des_pos_x" value="$(arg des_x)" />
    <param name="des_pos_y" value="$(arg des_y)" />

    <node name="follow_wall_server" pkg="crazyCmd" type="dd_follow_wall_server.py" output="screen"/>
    <node name="go_to_point_server" pkg="crazyCmd" type="dd_go_to_point_server.py" output="screen"/>
    <node name="bug2" pkg="crazyCmd" type="dd_bug2.py" output="screen"/>
</launch>